config.py

```diff
@@ -96,26 +130,26 @@ class Config(object):
         Queue('retry', Exchange('default'), routing_key='retry'),
         Queue('email-already-registered', Exchange('default'), routing_key='email-already-registered')
     ]
-    API_HOST_NAME = os.environ['API_HOST_NAME']
-    MMG_API_KEY = os.environ['MMG_API_KEY']
-    FIRETEXT_API_KEY = os.getenv("FIRETEXT_API_KEY")
-    LOADTESTING_NUMBER = os.getenv('LOADTESTING_NUMBER')
-    LOADTESTING_API_KEY = os.getenv("LOADTESTING_API_KEY")
-    CSV_UPLOAD_BUCKET_NAME = os.getenv("CSV_UPLOAD_BUCKET_NAME")
+    API_HOST_NAME = "http://localhost:6011"
+
+    CSV_UPLOAD_BUCKET_NAME = 'development-notifications-csv-upload'
```

Comment:
same with this being in Dev - i just think it's a bit nicer that if we're defining a new environment it's immediately obvious that we aren't using the dev bucket